src/rt-app.c

```diff
@@ -1017,8 +1015,7 @@ void *thread_body(void *arg)
 	{
 		log_notice("[%d] Locking pages in memory", data->ind);
 		ret = mlockall(MCL_CURRENT | MCL_FUTURE);
-		if (ret < 0) {
-			errno = ret;
+		if (ret != 0) {
```

Comment:
This change is fine, but seems to me a matter of style as this fn can only return 0 or -1. I don't mind the change though, it is clearer that 0 is the only acceptable result.

Reply:
I made this change to be consistent with the rest of the code that checks for (ret != 0)